src/tools/bin/plot_chargeConservation.py

```diff
@@ -52,55 +51,65 @@ def set_colorbar(cb):
         t.set_fontsize(16)
 
 
-def plotError(h5file, slice_pos=[0.5, 0.5, 0.5]):
+def plotError(file_pattern, slice_pos=[0.5, 0.5, 0.5], timestep=-1):
     """
-    read field data from hdf5 files
+    read field data from an openPMD file
     compute div(E) - rho/epsilon_0
     plot slices through simulation volume
 
     Parameters:
-    h5file: file name
-        file name to hdf5 data set from PIConGPU
+    file_pattern: file name
+         openPMD file series pattern e.g. simData_%%T.bp
 
     slice_pos: list of floats
         list of 3 floats to define slice position [0, 1]
         Default=[0.5, 0.5, 0.5]
+
+    timestep: selected timestep
+        simulation step used if file is an
+        openPMD file series pattern e.g. simData_%%T.bp
     """
-    # load hdf5 file
-    f = h5py.File(h5file, "r")
+    # load file
+    series = io.Series(file_pattern, io.Access.read_only)
 
     # read time step (python 2 and 3 save)
-    timestep = -1
-    for i in f['/data'].keys():
-        timestep = i
+    if timestep == -1:
+        for i in series.iterations:
+            timestep = i
```

Comment:
This loop approach is a python 2.x (outdated) version to get the last iteration of `series.iterations` to get the last iteration. 
Please use:
```python
*_, timestep = series.iterations
```
instead. 

If you want to keep the python 2.x compatibility, I would prefer the slightly cleaner approach:
```python
for timestep in series.iterations:
    continue
```

Reply:
will change to the first suggestion, python 2 is dead so there is no need to be compatible.

Reply:
I applied the change